backend/lcfs/web/api/fuel_code/services.py

```diff
@@ -176,6 +176,7 @@ async def create_fuel_code(self, fuel_code: FuelCodeCreateSchema) -> FuelCode:
         """
         Create a new fuel code.
         """
+        fuel_code.status = "Draft"
```

Comment:
It would be great if the draft string were replaced with `FuelCodeStatusEnum.Draft`